app/src/components/pages/divisionPage/divisionPage.tsx

```diff
@@ -61,101 +52,22 @@ export default function DivisionPage() {
-          </Heading>
-          <Spacer />
-          <Link
-            id={`row-see-more-items-${title}`}
-            type="standalone"
-            href={`#`}
-            aria-label={`See more items in ${title}`}
-            hasVisitedState
-            __css={{
-              display: { sm: "none", md: "inline" },
-              color: "ui.primary.link",
-              fontWeight: "500 !important",
-              alignItems: "center",
-              _hover: { textDecoration: "underline 1px dotted !important" },
-            }}
-          >
-            See more
-          </Link>
-        </Flex>
-        <SimpleGrid
-          columns={numColumns}
-          sx={{
-            gridTemplateColumns: `repeat(${numColumns}, minmax(0, 1fr))`,
-          }}
-        >
-          {mockItems.map((item, index) => {
-            const itemModel = new ItemCardModel(item);
-            return (
-              <ItemCard
-                key={index}
-                id={`item-${index}-${title}`}
-                item={itemModel}
-                isLargerThanLargeTablet={isLargerThanLargeTablet}
-              />
-            );
-          })}
-        </SimpleGrid>
+        <Heading level="h1" text={title} subtitle={data.summary} />
         <Link
-          id={`row-see-more-items-mobile-${title}`}
           type="standalone"
-          href={`#`}
-          aria-label={`See more items in ${title}`}
-          className="smlink"
-          hasVisitedState
-          __css={{
-            display: { sm: "flex", md: "none" },
-            fontWeight: "regular",
-            justifyContent: "flex-end",
-            marginTop: "s",
-            alignItems: "center",
-            "& svg": {
-              marginTop: "1px",
-            },
-          }}
+          target="_blank"
+          href={data.nyplLink}
+          style={{ width: "fit-content" }}
         >
-          See more
+          <span> Contact info and more </span>
         </Link>
       </Box>
+
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      <Heading level="h2" size="heading3">
-        {`Collections in the ${title}`}
-      </Heading>
-      <SimpleGrid
-        columns={numColumns}
-        sx={{
-          gridTemplateColumns: `repeat(${numColumns}, minmax(0, 1fr))`,
-        }}
-      >
-        {mockCollections.map((collection: CollectionDataType, index) => {
-          const collectionModel = new CollectionCardModel(collection);
-          return (
-            <CollectionCard
-              key={index}
-              id={index}
-              slug={collectionModel.title}
-              collection={collectionModel}
-              isLargerThanLargeTablet={isLargerThanLargeTablet}
-            />
-          );
-        })}
-      </SimpleGrid>
+      <ItemLane data={data} />
+
+      <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
+      <CollectionsGrid data={data} updateCollections={updateCollections} />
```

Comment:
The `updateCollections` function is not being used and the prop is not declared in `CollectionsGrid`. Remove it.

Reply:
more scalpels to remove from the patient, sorry about that.